crates/static-analysis-kernel/src/analysis/ddsa_lib/js/violation.rs

```diff
@@ -6,23 +6,22 @@ use crate::analysis::ddsa_lib::common::{
     get_field, get_optional_field, iter_v8_array, v8_type_from, DDSAJsRuntimeError, Instance,
 };
 use crate::analysis::ddsa_lib::js::fix::{Fix, FixConverter};
+use crate::analysis::ddsa_lib::js::region::{CodeRegion, CodeRegionConverter};
 use crate::analysis::ddsa_lib::v8_ds::V8Converter;
 use crate::model::rule::{RuleCategory, RuleSeverity};
 use crate::model::violation;
-use common::model::position::Position;
+use common::model::position;
 use deno_core::v8;
 use deno_core::v8::HandleScope;
 use std::marker::PhantomData;
 
 /// A representation of a JavaScript `Violation` class instance.
 #[derive(Debug, Clone, Eq, PartialEq, Hash)]
 pub struct Violation<T> {
-    pub start_line: u32,
-    pub start_col: u32,
-    pub end_line: u32,
-    pub end_col: u32,
```

Comment:
Why removing this? It's clear - it's the section that is annotated and shown. The "regions" should just be something that contains the whole annotations but we should not touch the existing part.